snyk/client.py

```diff
@@ -82,14 +89,28 @@ def request(
             raise SnykHTTPError(resp)
         return resp
 
-    def post(self, path: str, body: Any, headers: dict = {}) -> requests.Response:
-        url = f"{self.api_url}/{path}"
+    def post(
+        self,
+        path: str,
+        body: Any,
+        headers: dict = {},
```

Comment:
What version of Python do you use?  The type `dict` is allowed only from 3.9.  I saw in the `pyproject.toml` file that the allowed versions of Python are from version `3.7` and up.  If this is still the case I suggest using `Dict` instead.

Reply:
This was previously in the project, I can change it, but I need to make sure every instance gets changed.

Reply:
I'm not suggesting to upgrade.  But if we still want to support Python 3.7 we need to change line 96 to be something like:
```python
headers: Dict = {}. # note the capital D.
```